File: javascript/1117.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var validas=0;
var notaAtual=0
var media=0

while(validas != 2){
    notaAtual=parseFloat(lines.shift())
    if(notaAtual >= 0 && notaAtual <= 10){
        media+=notaAtual
        validas++
    }else{
        console.log("nota invalida")
    }
}
media/=2
console.log(`media = ${media.toFixed(2)}`)

File: C++/1117.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    double nota1,nota2,media;
    do{
        cin >> nota1;
        
        if (nota1<0 || nota1>10)
            cout << "nota invalida" << endl;
    }while (nota1<0 || nota1>10); 
    do{
        cin >> nota2;
        if (nota2<0 || nota2>10)
        cout << "nota invalida" << endl;
    }while (nota2<0 || nota2>10); 
    media = (nota1+nota2)/2;
    cout << "media = " << fixed << setprecision(2) << media << endl;
    return 0;
}